packages/antd/src/PollingUpdaterAnt.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react';
+import { Popover, Spin, Alert } from 'antd';
+import { observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { t } from '@moxb/moxb';
+import { renderUIFragment, UIFragment } from './not-antd';
+
+/**
+ * A callback type that will be passed to the update function
+ */
+export type PollingUpdaterCallback = (errorMessage: string | undefined, content: UIFragment | undefined) => void;
+
+interface PollingUpdaterImpl {
+    /**
+     * How many seconds between the updates?
+     */
+    readonly updateFrequency: number;
+
+    /**
+     * Don't show the pop-up window, with information about update times. (Default: false - show the popup)
+     */
+    noPopup?: boolean;
+
+    /**
+     * Disable for mechanism for for triggering an update by clicking. (Default: false - updates are enabled)
+     */
+    noUpdateOnClick?: boolean;
+
+    /**
+     * What should be in the title, when the popup is shown?
+     */
+    title: string;
+
+    /**
+     * This will be called to trigger an update
+     */
+    update(callback: PollingUpdaterCallback): void;
+}
+
+interface PollingUpdaterProps {
+    operation: PollingUpdaterImpl;
+}
+
+/**
+ * The PollingUpdaterAnt component displays some content that is periodically updated.
+ *
+ * The rate of updates (polling frequency) is configurable.
+ * There is a pop-up, which explains how much time is left till the next update.
+ * Immediate update is also possible, when clicking on the content.
+ */
+@observer
+export class PollingUpdaterAnt extends React.Component<PollingUpdaterProps> {
+    @observable protected _pending = false;
+    @observable protected _content: UIFragment | undefined;
+    @observable protected _failed = false;
+    @observable protected _errorMessage: string | undefined;
+    @observable protected _lastUpdate: number | undefined;
+    @observable protected _updateText: string | undefined;
```

Comment:
`@observable protected _updateText?: string;` the `?` is a shortcut for `| undefined`